tests/mtgjson5/test_price_parser.py

```diff
@@ -57,7 +57,7 @@ def test_card_kingdom_provider():
     ]
 
     assert len(expected_responses) == len(actual_responses)
-    for expected, actual in zip(actual_responses, expected_responses):
```

Comment:
"tests: fix order of parameters"

(ideally this should be squashed in the commit the broke it if it's in this PR)